source/command-line-options.txt

```diff
@@ -33,18 +35,75 @@ Use the following syntax to start |compass| and set a configuration option:
 Options
 -------
 
-You can set the following configuration options using the command line:
+You can set these options when using the command line:
+
+.. option:: --autoUpdates
+  
+   Enable :ref:`automatic updates <enable-automatic-updates>`.
```

Comment:
I think it’s worth mentioning that:

- Some of these options override values that are otherwise set in the settings (formerly privacy) panel
- For boolean options a `--no` prefix can be used to negate them, e.g. `--no-autoUpdates`

Otherwise `--autoUpdates` is not a particularly useful option on its own, since it’s set by default and can also be disabled from the settings panel directly.

Reply:
I've updated the options that accept boolean values to document the `--no-` versions as well. 

I've noted that setting options on the command line will override values set in **Settings**.